README.md

```diff
@@ -115,4 +115,7 @@ ember try ember-data-canary
 ### Running the FireBlog demo app
 
 * `ember server`
-* Visit your app at http://localhost:4200.
+* Visit your app at [http://localhost:4200][].
```

Comment:
Thanks for this. But we've been using the inline link syntax throughout this file:

e.g.

```
See the [Firebase documentation](https://firebase.com/docs/web/libraries/ember/guide.html#section-without-ember-cli) for instructions on getting started.
```